service/controller/chart/v1/resource/release/create.go

```diff
@@ -70,9 +71,30 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 			}
 		}()
 
-		// We need to pass the ValueOverrides option to make the install process
-		// use the default values and prevent errors on nested values.
-		err = r.helmClient.InstallReleaseFromTarball(ctx, tarballPath, ns, helm.ReleaseName(releaseState.Name), helm.ValueOverrides(releaseState.ValuesYAML))
+		ch := make(chan error)
+
+		// We create the helm release but with a short timeout so we don't
+		// block reconciling other CRs. This gives time to make the port
+		// forwarding connection to the Tiller API.
+		//
+		// If we do timeout the install will continue in the background.
+		// We will check the progress in the next reconciliation loop.
+		go func() {
+			// We need to pass the ValueOverrides option to make the install process
+			// use the default values and prevent errors on nested values.
+			err = r.helmClient.InstallReleaseFromTarball(ctx, tarballPath, ns, helm.ReleaseName(releaseState.Name), helm.ValueOverrides(releaseState.ValuesYAML))
+			close(ch)
+		}()
+
+		select {
+		case <-ch:
+			// Fall through.
+		case <-time.After(3 * time.Second):
```

Comment:
We use time.After and not context.WithTimeout because it doesn't work with our k8sportforward lib.

The timeout is short but many of our simple charts install quickly and then there is no change in behavior.